examples/ollama/Aspire.CommunityToolkit.Hosting.Ollama.AppHost/Program.cs

```diff
@@ -1,13 +1,18 @@
 var builder = DistributedApplication.CreateBuilder(args);
 
-var ollama = builder.AddOllama("ollama", port: null)
-    .AddModel("phi3")
-    .WithDefaultModel("phi3")
+var ollama = builder.AddOllama("ollama", port: 11434)
+    .AddModel("phi3.5")
+    .WithDefaultModel("phi3.5")
     .WithOpenWebUI();
 
 builder.AddProject<Projects.Aspire_CommunityToolkit_Hosting_Ollama_Web>("webfrontend")
     .WithExternalHttpEndpoints()
     .WithReference(ollama)
-    .WithEnvironment("ollama:model", "phi3");
+    .WithEnvironment("ollama:model", "phi3.5");
```

Comment:
There's no need to specify the model as an environment variable, we inject all model names as environment variables to the project/resource that it's referenced to:

![Environment variables for Ollama models](https://github.com/user-attachments/assets/d052d816-c185-4ffa-9c27-fae608be7811)

I do need to document that...